google_guest_agent/network/manager/netplan_linux.go

```diff
@@ -274,6 +277,16 @@ func (nd networkdNetplanDropin) write(n netplan, iface string) error {
 	return nil
 }
 
+// shouldUseDomains returns true if interface index is 0.
+func shouldUseDomains(idx int) *bool {
+	trueVal := true
```

Comment:
This can be simpler, can we just do something like `val := idx ==0; return &val`?

Reply:
Done